src/panels/lovelace/card-features/hui-select-options-card-feature.ts

```diff
@@ -126,6 +130,7 @@ class HuiSelectOptionsCardFeature
         hide-label
         .label=${this.hass.localize("ui.card.select.option")}
         .value=${stateObj.state}
+        .options=${options}
```

Comment:
`options` will be recreated on ever render it seems, that means we are calling `layoutOptions` and `selectByValue` every render too?

Reply:
I didn't notice that, could just memoize it?